### PR TITLE
Make frontend use /data/environment route

### PR DESCRIPTION
**Pull request by chihuahua**
Previously, the frontend displayed the logdir under the run selector.
This change makes that area display either the logdir or the SQL URI
depending on which is used.

image

### CODE REVIEW

File: tensorboard/components/tf_runs_selector/tf-runs-selector.html [hunk 1]
```diff
@@ -123,16 +123,16 @@ <h3 id="tooltip-help" class="tooltip-container">Runs</h3>
...
       runs: Array,
-      logdir: {
+      dataLocation: {
         type: String,
         notify: true,
       },
-      // This is the potentially clipped portion of the logdir we show at the bottom of the sidebar.
-      _clippedLogdir: {
+      // This is the potentially clipped portion of the dataLocation we show at the bottom of the sidebar.
+      _clippeddataLocation: {
```
Comment (nfelt) [inline tensorboard/components/tf_runs_selector/tf-runs-selector.html]: nit: capitalize Data

File: tensorboard/components/tf_runs_selector/tf-runs-selector.html [hunk 2]
```diff
@@ -160,26 +161,26 @@ <h3 id="tooltip-help" class="tooltip-container">Runs</h3>
...
     },
-    _openLogdirDialog: function(event) {
+    _opendataLocationDialog: function(event) {
       event.preventDefault();
-      this.$$('#logdir-dialog').open();
+      this.$$('#data-location-dialog').open();
     },
-    _shouldShowExpandLogdirButton(logdir, _logdirClipLength) {
-      return logdir && logdir.length > _logdirClipLength;
+    _shouldShowExpanddataLocationButton(dataLocation, _dataLocationClipLength) {
```
Comment (nfelt) [inline tensorboard/components/tf_runs_selector/tf-runs-selector.html]: nit: capitalize Data

File: tensorboard/components/tf_runs_selector/tf-runs-selector.html [hunk 3]
```diff
@@ -160,26 +161,26 @@ <h3 id="tooltip-help" class="tooltip-container">Runs</h3>
...
+        // Clip the dataLocation to avoid blocking the runs selector. Let the
+        // user view a more full version of the dataLocation.
+        return this._breakString(dataLocation.substring(0, dataLocationClipLength));
       } else {
-        return this._breakString(logdir);
+        return this._breakString(dataLocation);
       }
     },
-    _openLogdirDialog: function(event) {
+    _opendataLocationDialog: function(event) {
```
Comment (nfelt) [inline tensorboard/components/tf_runs_selector/tf-runs-selector.html]: nit: capitalize Data

File: tensorboard/components/tf_tensorboard/tf-tensorboard.html [hunk 4]
```diff
@@ -478,7 +478,7 @@ <h3>There’s no dashboard by the name of “<tt>[[_selectedDashboard]]</tt>.”
           value: "not yet loaded",
         },
 
-        _logdir: {
+        _data_location: {
```
Comment (nfelt) [inline tensorboard/components/tf_tensorboard/tf-tensorboard.html]: prefer `_dataLocation` to match the lowerCamelCase of the other properties

File: tensorboard/components/tf_runs_selector/tf-runs-selector.html (hunk 1 above)
Comment (chihuahua) [reply tensorboard/components/tf_runs_selector/tf-runs-selector.html]: Done.

File: tensorboard/components/tf_runs_selector/tf-runs-selector.html (hunk 2 above)
Comment (chihuahua) [reply tensorboard/components/tf_runs_selector/tf-runs-selector.html]: Done.

File: tensorboard/components/tf_runs_selector/tf-runs-selector.html (hunk 3 above)
Comment (chihuahua) [reply tensorboard/components/tf_runs_selector/tf-runs-selector.html]: Done.

**Pull request merged**